fusor-ember-cli/app/routes/review/installation.js

```diff
@@ -27,6 +27,38 @@ export default Ember.Route.extend({
             controller.set('openstackProfiles', flavors);
         });
     }
+    if (!model.get('isStarted')) {
+        var self = this;
+        var deployment = self.modelFor('deployment');
+        var token = Ember.$('meta[name="csrf-token"]').attr('content');
+
+        Ember.$.ajax({
```

Comment:
@daviddavis, yes, please use ic-ajax here since there is "ember aware" and will use the server mocks if they exist. Plus, ic-ajax `request()` method uses regular Javascript Promises.